serde/src/test/kotlin/com/amazon/ionhiveserde/objectinspectors/IonStructToMapObjectInspectorTest.kt

```diff
@@ -59,26 +60,43 @@ class IonStructToMapObjectInspectorTest {
         assertNull(subject.getMapValueElement(makeStruct(), null))
     }
 
+    @Test
+    fun getMapSize() {
+        assertEquals(3, subject.getMapSize(makeStruct()))
+    }
+
     @Test
     fun getMap() {
         val struct = makeStruct()
-
         val actual = subject.getMap(struct)
-        assertEquals(3, actual.size)
```

Comment:
Remove this line to be a separate unit test, see [getMapSize](https://github.com/amzn/ion-hive-serde/pull/91/commits/91736de6c0c65822c403af9c7c13e53643f2a329#diff-e277b22a3db092c7502625ccb36b032f85990fd78606a2c7becfb6684500b09bR64) above.